testsuite/MDAnalysisTests/analysis/test_msd.py

```diff
@@ -147,6 +152,15 @@ def test_random_walk_u_simple(self, random_walk_u):
         norm = np.linalg.norm(msd_rw.results.timeseries)
         val = 3932.39927487146
         assert_almost_equal(norm, val, decimal=5)
+        # nongaussian parameter may have negative values (above -2.5) and 
+        # postiive values, but converge to zero at long times.
+        assert_almost_equal(msd_rw.results.nongaussian_parameter[:20], 
```

Comment:
maybe `assert_allclose()` for new tests? https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html